source/slang/slang-ir-inst-defs.h

```diff
@@ -997,6 +992,9 @@ INST(HighLevelDeclDecoration,               highLevelDecl,          1, 0)
         /// Recognized by SPIRV-emit pass so we can emit a SPIRV `Block` decoration.
     INST(SPIRVBlockDecoration, spvBlock, 0, 0)
 
+        // Stores flag bits of which memory qualifiers an object has
+    INST(MemoryQualifierCollectionDecoration, MemoryQualifierCollectionDecoration, 1, 0)
```

Comment:
I don't know where the keyword "collection" came from.
But judging by the implementation here, it looks more like "set".
Can you consider renaming "collection" to "set"?